daemon/src/auto_rollover.rs

```diff
@@ -139,10 +139,7 @@ where
 {
     async fn started(&mut self, ctx: &mut xtra::Context<Self>) {
         let fut = ctx
-            .notify_interval(
-                rollover_taker::MAX_ROLLOVER_DURATION + Duration::from_secs(60),
-                || AutoRollover,
-            )
+            .notify_interval(Duration::from_secs(5 * 60), || AutoRollover)
```

Comment:
❓ Why the magic number `5`?

Reply:
This just restores the previous functionality. `MAX_ROLLOVER_DURATION` was 4 minutes, I just made this not depend on this constant anymore because the internals now work differently.